README.md

```diff
@@ -1,43 +1,71 @@
 # airtable-schema-generator
 
-An NPM package with a script to download an Airtable schema, and generate schema file, constants, and request helpers! 
+An NPM package designed to support React + Node apps that use Airtable as a backend!
 
 ## What it does
 
-This script open an electron app browser window and opens the Airtable API page for your base using your login credentials. Once the api page loads, the script generates 3 files: `schema.json`, `request.js`, and `schema.js`.
+After you install and setup the package, running `yarn generate-schema` (or `npm run generate-schema`), the script will scrape the Airtable website, download the latest schema for your app, and generate helper files. 
 
-The first file is a simplified JSON object representing your airtable base. The last two files work with the sample `airtable.js` file to create nice CRUD helpers and constants for a javascript project using Airtable. Copy over `airtable.js` to the same folder as the generated files to fully utilize the helpers. Note that the provided `airtable.js` example transforms records' column names to and from camel case and original airtable format for easy developing!
+The helper files establish abstraction barriers for the rest of your project to follow. At the lowest level is **`airtable.js`**, which contains the functions that directly interact with airtable. Next, comes **`request.js`**, which are CRUD functions generated for _every_ table in your Airtable base. Finally is **`schema.js`**, a source of truth for your airtable schema.
 
-Details on the design decisions behind the architecture can be found [here](https://www.notion.so/calblueprint/PP-Power-Airtable-Client-Side-6a1b6734af294ef88609a6d6d256ca3d).
+Besides organizing your code's interactions with airtable via helper functions, the package also makes Airtable records easier to work with! The file `schema.js` contains mappings from Airtable Column Names to a more javascript-y version of them. For example, "First Name" becomes "firstName". It's aware of linked records as well; "Project Group" (which might be a one-to-one linked record column) will become "projectGroupId". 
+
+In your codebase, you can use the javascript-y names, and the provided `airtable.js` file will automatically transform them to and from the actual Airtable column names!
```

Comment:
Personally I found it very helpful to look through `schema.js` and `request.js` to sanity-check what functions I was calling, so maybe could add a line here that they can?

And/or link to the Records Transformation section!